state/presence/pingbatcher_test.go

```diff
@@ -0,0 +1,274 @@
+// Copyright 2017 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package presence_test
+
+import (
+	"fmt"
+	"time"
+
+	// 	"github.com/juju/errors"
```

Comment:
indeed, thanks for the catch. (my editor defaults to hiding the import group and when starting a new test file you're never quite sure how many of them you're going to need)